docs/cases/cases-overview.asciidoc

```diff
@@ -31,7 +31,11 @@ image::images/cases-home-page.png[Case UI Home]
 Open a new case to keep track of security issues and share their details with
 colleagues.
 
-. Go to *Cases* -> *Create new case*.
+. Go to the *Cases* page and click *Create case*. If no cases exist, the Cases table will be empty and you'll be prompted to create one by clicking the *Create case* button inside the table. 
```

Comment:
On other pages, the format would be more like this. But the way you've written it is also perfectly readable, so just pointing this out for possible consistency.
```suggestion
. Go to *Detect* -> *Cases*, then click *Create case*. If no cases exist, the Cases table will be empty and you'll be prompted to create one by clicking the *Create case* button inside the table. 
```

Reply:
I actually like this better _because_ it's consistent with the format our docs typically follow!